tasks/test.py

```diff
@@ -1168,3 +1170,115 @@ def get_modified_files(ctx):
 
     modified_files = ctx.run(f"git diff --name-only {last_main_commit}", hide=True).stdout.splitlines()
     return modified_files
+
+
+@task
+def send_unit_tests_stats(_, job_name):
```

Comment:
:question: Why is the job name (it's a bit modified but it's very similar) provided as an argument instead of fetching it through the environment?

Reply:
Because the job that will execute this invoke task is `stats-fast-tests-*-x64-py3` so the `CI_JOB_NAME` variable would be `stats-fast-tests-*-x64-py3` while I'd like it to be the name of the job that actually run the tests